### DISCUSSION TITLE
using retrieve endpoint with postman

### DISCUSSION THREAD

**Discussion by tmartinscopo** [serenity-is/Serenity#7255] [General]
Hello
I'm trying to use endpoint retrieve through postman. How do I pass the entityId as a parameter?
![image](https://github.com/user-attachments/assets/d9a6b933-f848-4ec5-bb86-92b826a4f12d)
Thanks for you help

**Comment by tmartinscopo** [comment]
Body request was missing!

**Comment by muhammedyaman** [comment]
@tmartinscopo  can you share your postman collection please? I need a sample.
